packages/yourloops/test/integration/mock/mockPatientAPI.ts

```diff
@@ -25,158 +25,218 @@
     firstName: 'Pending',
     fullName: 'Pending Patient',
     lastName: 'Patient',
-    patient: { birthday: '1980-01-01T10:44:34+01:00', diagnosisType: 'type1' },
-    privacyPolicy: { acceptanceTimestamp: '2021-05-22', isAccepted: true },
-    termsOfUse: { acceptanceTimestamp: '2021-05-22', isAccepted: true },
-    trainingAck: { acceptanceTimestamp: '2022-10-11', isAccepted: true }
-  } as Profile,
-  settings: null,
-  preferences: { displayLanguageCode: 'en' },
-  invitationStatus: UserInvitationStatus.pending,
-  email: 'ylp.ui.test.patient29@diabeloop.fr',
-  idVerified: false,
-  unreadMessages: 0,
-  alarms: {
-    timeSpentAwayFromTargetRate: 0,
-    timeSpentAwayFromTargetActive: false,
-    frequencyOfSevereHypoglycemiaRate: 0,
-    frequencyOfSevereHypoglycemiaActive: false,
-    nonDataTransmissionRate: 0,
-    nonDataTransmissionActive: false
+    sex: 'M'
   },
-  monitoring: null
+  defaultSettings,
+  defaultMetadata,
+  defaultAlarm
+)
+
+export const buildTeamMemberFromPatient = (patient: Patient): ITeamMember => {
+  return {
+    userId: patient.userid,
+    teamId: patient.teams[0].teamId,
+    role: TeamMemberRole.patient,
+    profile: {
+      email: patient.profile.email,
+      firstName: patient.profile.firstName,
+      fullName: patient.profile.fullName,
+      lastName: patient.profile.lastName,
+      patient: { birthday: '1980-01-01T10:44:34+01:00', diagnosisType: 'type1' },
+      privacyPolicy: { acceptanceTimestamp: '2021-05-22', isAccepted: true },
+      termsOfUse: { acceptanceTimestamp: '2021-05-22', isAccepted: true },
+      trainingAck: { acceptanceTimestamp: '2022-10-11', isAccepted: true }
+    },
+    settings: null,
+    preferences: { displayLanguageCode: 'en' },
+    invitationStatus: patient.teams[0].status,
+    email: patient.profile.email,
+    idVerified: false,
+    unreadMessages: patient.metadata.hasSentUnreadMessages ? 1 : 0,
+    alarms: patient.alarms,
+    monitoring: patient.monitoring
+  }
 }
 
-export const removePatientMock = jest.spyOn(PatientAPI, 'removePatient').mockResolvedValue(undefined)
-export const mockPatientAPI = () => {
-  jest.spyOn(PatientAPI, 'getPatients').mockResolvedValue([monitoredPatient, unmonitoredPatient, monitoredPatientTwo, pendingPatient])
+export const monitoredPatientAsTeamMember: ITeamMember = buildTeamMemberFromPatient(monitoredPatient)
+export const unmonitoredPatientAsTeamMember: ITeamMember = buildTeamMemberFromPatient(unmonitoredPatient)
+const monitoredPatientTwoAsTeamMember: ITeamMember = buildTeamMemberFromPatient(monitoredPatientTwo)
+export const pendingPatientAsTeamMember: ITeamMember = buildTeamMemberFromPatient(pendingPatient)
+
+export const mockPatientAPIForPatients = () => {
+  jest.spyOn(PatientApi, 'getPatients').mockResolvedValue([monitoredPatientAsTeamMember, unmonitoredPatientAsTeamMember, monitoredPatientTwoAsTeamMember, pendingPatientAsTeamMember])
+}
+export const mockPatientAPIForHcp = () => {
```

Comment:
There is a naming inconsistency, because 2 ways of writing acronyms in PascalCase are mixed (`API` and `Hcp`). It would be best to choose only one and apply it everywhere, but especially in the same variable name 😄 

I personally prefer having only first letter in uppercase and the others in lowercase, as it is stated [here](https://cis300.cs.ksu.edu/appendix/style/naming/#:~:text=Pascal%20case%3A%20Multiple%20words%20are,%2C%20AverageAge%20%2C%20ContextBoundObject%20%2C%20RgbCode%20.):
> If acronyms of 3 or more letters are included, only the first letter of the acronym is capitalized.